tck/faces22/ajax/src/test/java/ee/jakarta/tck/faces/test/servlet30/ajax/Issue2162IT.java

```diff
@@ -15,6 +15,12 @@
  */
 
 package ee.jakarta.tck.faces.test.servlet30.ajax;
+/**
+ * @see PreRenderViewEvent
+ * @see AjaxBehavior
+ * @see https://github.com/eclipse-ee4j/mojarra/issues/2166
+ */
+
```

Comment:
I guess this change was unintentional?

Reply:
I have to check that... will do tomorrow.